apps/arweave/src/ar_packing_server.erl

```diff
@@ -528,18 +528,16 @@ calling_function([_, {_, _, _, _}|[{Module, Function, Arity, _}|_]]) ->
 calling_function(_) ->
     "unknown".
 
-record_packing_request(Type, Packing, From) when
-	is_atom(Type), is_atom(Packing), is_list(From) ->
+record_packing_request(Type, RequestedPacking, StoredPacking, From) ->
+	Type2 = case RequestedPacking == StoredPacking of
```

Comment:
Sometimes StoredPacking passed is `unpacked`, sometimes - `unpack`, is it intentional? What do you think about only tracking the actual packings and unpackings (where randomx_encrypt/decrypt_chunk's are called)?

Reply:
Hm, probably not intentional. I'll review! And I agree tracking actual packing/unpacking is best - the old logic tracked everything without distinguishing whether an encrypt/decrypt actually happened, this "fix" intended to update the logic to flag when an encrypt/decrypt actually happened. I can review again and maybe simply remove the tracking for no-ops rather than tag them as no-ops.

Reply:
Fixed!